cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,37 @@
-describe('', () => {
-  before(() => {
+import { faker } from '@faker-js/faker';
 
+describe('Article Flow', () => {
+  beforeEach(() => {
+    cy.visit('https://conduit.mate.academy/');
   });
 
-  it('', () => {
+  it('Create and Delete Article', () => {
+    const user = {
+      username: faker.internet.userName(),
+      email: faker.internet.email().toLowerCase(),
+      password: '12345Qwert!'
+    };
 
-  });
+    // Registering a new user
+    cy.get('a').contains('Sign up').click();
```

Comment:
Your task is to check next flow:

Create the article:
use login() command to sign in before the test.